client/reader/sidebar/reader-sidebar-tags/index.jsx

```diff
@@ -33,29 +35,26 @@ export class ReaderSidebarTags extends Component {
 	}
 
 	followTag = ( tag ) => {
-		const subscription = TagStore.getSubscription( TagActions.slugify( tag ) );
-		if ( subscription ) {
```

Comment:
I deleted this because I couldn't figure out the case this was protecting from.  
If we knew that the user was subscribed to it then we wouldn't have showed them a follow button anyway